README.md

```diff
@@ -86,3 +86,25 @@ Create `.env.production.local` and set values as suggested by backend team. Then
 ```shell
 yarn build
 ```
+## Cypress Sanity Tests
+
+UI sanity tests are located in `cypress/integration` directory
+
+### How to run
+
+Build the app first and then run the tests:
+```shell
+yarn run cypress
+```
+
+### Skipping failing tests
+
+Most of the selectors are text-based and are prone to failure on UI change.
+If a tests fails due to change in the UI / selectors you can skip it by modifying the `*.spec.js` file in `cypress/integration` and adding `skip` attribute to `it()`.
+```javascript
+it.skip('Failing test', () => {})
+```
```

Comment:
I would propose a diffrent approach here. Let's add `data-testid` attributes on the elements, as recommended here:
[https://testing-library.com/docs/queries/bytestid/](https://testing-library.com/docs/queries/bytestid/)
and then, strip them from production build using
[https://www.npmjs.com/package/babel-plugin-jsx-remove-data-test-id](https://www.npmjs.com/package/babel-plugin-jsx-remove-data-test-id)

This way we ensure stability of the selectors (even when CSS classes change) and don't send the test mess to the production client.

Reply:
After consideration: please disregard this comment for this PR and create a story for the next PR when we'll tackle this issue, as the upcoming PR with TS migration will cause a lot of conflicts if you modify the frontend code now.

Reply:
Sounds great! I would have suggested this as the next step as well, since text selectors are not the best, and were the only thing that I could stick to, at the moment :)